### PR TITLE
[QRF-119] Add forbidden characters replacement for file names

### PR DESCRIPTION
**Pull request by aspirisen**
https://linear.app/prezly/issue/QRF-119/unable-to-load-attached-pdf-in-campaign-email

### CODE REVIEW

**Comment by aspirisen** [comment]
@kudlajz could you take a look

File: src/lib/UploadcareFile.ts [hunk 1]
```diff
@@ -68,4 +69,8 @@ export class UploadcareFile {
         mime_type: this.mimeType,
         size: this.size,
     });
+
+    private static replaceForbiddenSymbols = (filename: string) => {
```
Comment (kudlajz) [inline src/lib/UploadcareFile.ts]: Why do you need this method? 🤔 Can't you simply inline the `sanitizeFilename` line? Or just have it as a util function?

**Review by kudlajz — APPROVED** [review]
Looks good 👍

**Review by e1himself — APPROVED** [review]
Looks good :+1: 

I also agree with Lukas' remark on not needing the replacement function as a method.

**Pull request merged**